resources/groundwork/web.md

```diff
@@ -49,7 +49,64 @@ Have some technologies you enjoy using? See our [Contribution Guide](/CONTRIBUTI
+  - For build tools, [`Babel`](https://babeljs.io/docs/en/) transcompiles modern JavaScript and JSX into older versions for compatibility
+- Considerations
+  - Pro: React's popularity in the community means that there is no shortage of learning resources
+  - Con: React is less opinionated than other front-end frameworks like Vue or Angular and can become difficult to structurally maintain in the long run. React also uses a lot of "magic" and beginners diving deep into it without learning JavaScript fundamentals beforehand often find themselves struggling later
+
+- References
+  - [React documentation](https://reactjs.org/)
+  - [JavaScript resources](/resources/languages.md#javascript)
+  - [TypeScript resources](/resources/languages.md#typescript)
+
+**Backend**: Node.js
+
+- Frameworks and libraries
+  - [`Express.js`](https://expressjs.com/): a minimal web application framework used for building REST APIs
+  - [`Socket.io`](https://socket.io/): allows us to manage real-time event-based communication using web sockets
+- Tooling
+  - For productivity, [`nodemon`](https://nodemon.io/) automatically restarts the application upon changes detected
+- Considerations
+  - Pro: Node.js applications scale up well and are known for being performant. Like React, it is also backed by a large community and hence, learning resources are always available
+  - Con: Being a single-threaded environment, Node.js is not suitable for CPU-intensive applications (e.g. audio/video processing)
+- References
+  - [Node.js documentation](https://nodejs.org/en/)
+
+**Datastore**: MongoDB
+
+- Frameworks and libraries
+  - [`Mongoose`](https://mongoosejs.com/): a popular Object Data Modelling (ODM) library for MongoDB
+- Tooling
+  - For visualizing and manipulating data quickly, [`MongoDB Compass`](https://www.mongodb.com/products/compass) or [`Studio 3T`](https://studio3t.com/) can be installed on Mac, Linux, or Windows
+- Considerations
+  - Pro:
+    - Being a NoSQL database, MongoDB is good for non-structured or schemaless data. The usage of JSON-like documents makes it pair well with Node.js
+    - A [`Docker Image`](https://hub.docker.com/_/mongo) for MongoDB is available. If you have Docker installed, you can spin up a MongoDB container very easily without installing mongo locally
+    - A cloud-hosted MongoDB service, [`MongoDB Atlas`](https://www.mongodb.com/cloud/atlas) is available on AWS, Azure and Google Cloud. With this service, you can deploy, operate, and scale a MongoDB database in just a few clicks
+  - Con: MongoDB Atlas Free Tier only allows 10GB in/out of a cluster per week - see [Atlas M0 (Free Tier), M2, and M5 Limitations](https://docs.atlas.mongodb.com/reference/free-shared-limitations/)
+- References
+  - [MongoDB documentation](https://docs.mongodb.com/)
+
+**Other Tooling**:
+- For managing dependencies, [`npm`](https://www.npmjs.com/) is the default for Node.js but [`yarn`](https://yarnpkg.com/) is a good alternative
```

Comment:
I feel like `yarn` isn't worth bringing up - their primary difference is how they handle their lockfiles, which doesn't really matter to most launch pad projects since we can be pretty ad-hoc about our dependencies, and it's kind of an awkward thing to have to keep asking members to using if the default instructions they see everywhere are `npm ...` 🤔